blueocean-dashboard/src/main/js/creation/git/ConnectStep.jsx

```diff
@@ -0,0 +1,190 @@
+/**
+ * Created by cmeyers on 10/19/16.
+ */
+import React, { PropTypes } from 'react';
+import FlowStep from '../FlowStep';
+
+const CREDENTIAL_CHOICE = {
+    SSH: 'SSH',
+    SYSTEM_SSH: 'SYSTEM_SSH',
+    USER_PASS: 'USER_PASS',
+};
+
+/**
+ * Component that accepts repository URL and credentials to initiate
+ * creation of a new pipeline.
+ */
+export default class ConnectStep extends React.Component {
```

Comment:
Should this be called "GitConnectStep"? 

This is in dashboard know, but I think @imeredith has proposed before it could one day migrate to another existing plugin - still the intention?